interfaces/led_driver.yaml

```diff
@@ -0,0 +1,16 @@
+description: >-
+  This interface provides a command for steering LEDs
+cmds:
+  set_led_state:
+    description: This command changes the color and brightness of LEDs
```

Comment:
Maybe this command can take an array of LED states with their brightness and potentially additional identifying information like an explicit id. Otherwise you can only ever control one led with one interface implementation

Reply:
I changed it, so it also adds connector_id. Therefore we don't have to keep track of all the LED states and change only the one we want